components/ResourcePage.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+/**
+ * Retrieves all resources of a specified resource type from the test server than returns a JSX element of
+ * the complete JSON body retrieved from the request
+ * @param props includes resourceLink which is a string specifying which resource type to retrieve from the server
+ * @returns JSX element
+ */
+const ResourcePage = (props) => {
```

Comment:
I know that this component will/has changed, but just so we don't get any TypeScript error, can we just add `any` here for now so this file doesn't throw any errors during an `npm run build`? 
```suggestion
const ResourcePage = (props: any) => {
```

Reply:
I just deleted the file b/c we don't  and won't use this component